docs/example/basic.rst

```diff
@@ -192,21 +192,25 @@ multiple PyPI servers, using ``PIP_EXTRA_INDEX_URL`` environment variable:
 
 .. code-block:: ini
 
-    [tox]
+    [testenv]
     setenv =
         PIP_EXTRA_INDEX_URL = https://mypypiserver.org
-
-    [testenv]
     deps =
         # docutils will be installed directly from PyPI
         docutils
         # mypackage missing at PyPI will be installed from custom PyPI URL
         mypackage
 
 This configuration will install ``docutils`` from the default
-Python PYPI server and will install the ``mypackage`` from
+Python PyPI server and will install the ``mypackage`` from
 our index server at ``https://mypypiserver.org`` URL.
 
+.. warning::
+
+  Using an extra PyPI index as a fallback for installing private packages may cause security issues.
```

Comment:
```suggestion
  Using an extra PyPI index for installing private packages may cause security issues.
```

I would remove the `as a fallback` as the intention would be to use the private server as the main server for private packages, which then causes the security issues when somebody registers the package on the public PyPI server.